README.md

```diff
@@ -72,8 +72,9 @@ Configure these options as parameters for the `init()` method or as environment
 | accessToken             | SIGNALFX_ACCESS_TOKEN               |           | The optional organization access token for trace submission requests |
 | enabled                 | SIGNALFX_TRACING_ENABLED            | true      | Whether to enable the tracer. |
 | debug                   | SIGNALFX_TRACING_DEBUG              | false     | Enable debug logging in the tracer. |
-| logInjection            | SIGNALFX_LOGS_INJECTION             | false     | Enable automatic injection of trace IDs in logs for supported logging libraries. |
 | tags                    | SIGNALFX_SPAN_TAGS                  | {}        | Set global tags that should be applied to all spans. Format for the environment variable is `key1:val1,key2:val2`. |
+| logInjection            | SIGNALFX_LOGS_INJECTION             | false     | Enable automatic injection of trace IDs in logs for supported logging libraries. |
+| logInjectionTags        | SIGNALFX_LOG_INJECTION_TAGS         | ['service.name'] | If log injection is on, these tags will be injected to log context. |
```

Comment:
Maybe keep the same prefix in both (`SIGNALFX_LOGS_`)? Not a big issue.

Reply:
At first `SIGNALFX_LOGS_INJECTION_TAGS` sounded funky, but indeed. Switched to this version for consistency